auth/dao/auth-dao.go

```diff
@@ -14,16 +14,24 @@ type DAO struct {
 	DB *sql.DB
 }
 
-// AuthCreateRequest contains the information required to create a new authorized user
-type AuthCreateRequest struct {
+// PlaintextAuth contains the information retrieved from an end user
+type PlaintextAuth struct {
```

Comment:
@smithwjv What do you think about these types? I've gone away from the `*Request` and `*Response` types because they're always the same - but distinguished between plaintext and hashed so that it's clear what types we're storing/retrieving?